content/docs/deploy/core/upgrading.mdx

```diff
@@ -13,6 +13,52 @@ sidebar_position: 10
 
 # Upgrade guide
 
+## 0.24.0
+
+### Breaking
+
+#### Set Authorization Header
+
+The deprecated `set_authorization_header` configuration setting has been removed. You can use the [Set Request Headers](/docs/reference/routes/headers#set-request-headers) setting to pass IdP tokens to upstream services in any header.
+
+#### Base64-encoded Certificates
```

Comment:
This is a little subtle: if I understand correctly, base64-encoded certificates are still supported under the `certificate` key, just not the `certificates` list. Is there some way we can clarify this heading, to avoid alarming anyone using the supported `certificate` setting?